cmd/blockchaincmd/add_validator.go

```diff
@@ -372,6 +364,7 @@ func CallAddValidator(
 	remainingBalanceOwnerAddr string,
 	disableOwnerAddr string,
 	sc models.Sidecar,
+	rpcURL string,
```

Comment:
What's the point of passing this and other global variables (flags in this case) to this function as arguments as opposed to just using them directly?

Reply:
the goal is to be able to call this command from another command, and pass to it only the needed values. this concept is best represented in the tokentransferrer deploy cmd, here the implementation is partial. 
secondary goal is to avoid using global variables at all: refactoring of common code, and different dev styles, showed that the risk to use/set global variables among several functions is possible and happens, so, to avoid secondary effects, and difficult to follow up code flow, it is best to just not use them.